kv/txn_correctness_test.go

```diff
@@ -55,6 +55,10 @@ func setCorrectnessRetryOptions(lSender *LocalSender) {
 	})
 }
 
+type runner interface {
```

Comment:
This type allows `client.KV` or `client.Txn` to be used below. Perhaps it should be defined in the `client` package.